### PR TITLE
Moving Azure Front Door steps back

### PR DESCRIPTION
**Pull request by v-fearam**
Moving Azure front Door to Share Resource
Adding WAF to AFD
Fixing after a complete deployment end-to-end

### CODE REVIEW

File: certificate-generation/test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Microsoft
```
Comment (ferantivero) [inline certificate-generation/test.txt:1]: do we need to distribute this one?

File: shared-svcs-stamp.json [hunk 2]
```diff
@@ -67,7 +73,10 @@
         "logAnalyticsWorkspaceName": "[concat('la-', variables('subRgUniqueString'))]",
         "defaultAcrName": "[concat('acraks', variables('subRgUniqueString'))]",
 
-        "acrPrivateDnsZonesName": "privatelink.azurecr.io"
+        "acrPrivateDnsZonesName": "privatelink.azurecr.io",
+
+        "frontDoorName": "[concat('multicluster-fd-', variables('subRgUniqueString'))]",
```
Comment (ferantivero) [inline shared-svcs-stamp.json:78]:
```suggestion
        "frontDoorName": "[concat('bicycle', variables('subRgUniqueString'))]",
```

**Review by ferantivero — CHANGES_REQUESTED** [review]

File: certificate-generation/test.txt (hunk 1 above)
Comment (v-fearam) [reply certificate-generation/test.txt:1]: My fault, deleting. Good catch, thanks.

File: shared-svcs-stamp.json (hunk 2 above)
Comment (v-fearam) [reply shared-svcs-stamp.json]: done

File: certificate-generation/test.txt (hunk 1 above)
Comment (v-fearam) [reply certificate-generation/test.txt]: done

**Review by ferantivero — APPROVED** [review]

**Pull request merged**